doc/man3/SSL_CTX_set_options.pod

```diff
@@ -170,6 +170,12 @@ RFC7366 Encrypt-then-MAC option on TLS and DTLS connection.
 If this option is set, Encrypt-then-MAC is disabled. Clients will not
 propose, and servers will not accept the extension.
 
+=item SSL_OP_NO_RENEGOTIATION
+
+Disable all renegotiation. Do not accept renegotiation requests.
+
+This also disables KeyUpdate messages in TLSv1.3.
```

Comment:
I don't think it is desirable. There are plenty of reasons to disable reneg. I can't think of any to disabled KeyUpdate.